go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/gorilla/handlers v1.5.1
 	github.com/gorilla/mux v1.8.0
 	github.com/hashicorp/golang-lru v0.5.4
+	github.com/klauspost/compress v1.15.12
```

Comment:
Current version is v1.16.5

Several releases in v1.16 contain substantial fixes for zstd. Ex: 

- https://github.com/klauspost/compress/releases/tag/v1.16.4
- https://github.com/klauspost/compress/releases/tag/v1.16.5